src/plugins/targetinfo/MQ2TargetInfo.cpp

```diff
@@ -79,6 +81,13 @@ int Target_AggroPctSecondaryLabel_TopOffsetOrg = 33;
 int Target_AggroPctPlayerLabel_BottomOffsetOrg = 47;
 int Target_AggroNameSecondaryLabel_BottomOffsetOrg = 47;
 int Target_AggroPctSecondaryLabel_BottomOffsetOrg = 47;
+MQColor	targetMeleeRng = MQColor(0, 0, 255);
```

Comment:
Should use a more descriptive name to indicate that this is a color, like `targetMeleeRngColor`